packages/riipen-ui/src/components/Notice.test.jsx

```diff
@@ -0,0 +1,109 @@
+import { mount } from "enzyme";
+import toJson from "enzyme-to-json";
+import React from "react";
+
+import Notice from "./Notice.jsx";
+
+describe("<Notice>", () => {
+  it("renders without errors", () => {
+    let error;
+    try {
+      mount(<Notice />);
+    } catch (e) {
+      error = e;
+    }
+    expect(error).toEqual(undefined);
+  });
+
+  it("renders correct snapshot", () => {
+    const wrapper = mount(<Notice />);
+
+    expect(toJson(wrapper)).toMatchSnapshot();
+  });
+
+  describe("default props", () => {
+    it("sets correct default props", () => {
+      const defaultProps = new Notice().type.defaultProps;
+
+      const wrapper = mount(<Notice />);
+
+      const component = wrapper.find("Notice");
+      expect(component.props().color).toEqual(defaultProps.color);
+    });
+  });
+
+  describe("children prop", () => {
+    it("renders given children", () => {
+      const child = <h5>hi hi</h5>;
+
+      const wrapper = mount(<Notice children={child} />);
```

Comment:
The usual way of doing children is `<Notice>{child}</Notice>`. I think our test should reflect that.